councilmatic/settings_deployment.py.example

```diff
@@ -43,6 +43,9 @@ CACHES = {
 # Set this to flush the cache at /flush-cache/{FLUSH_KEY}
 FLUSH_KEY = 'super secret junk'
 
+# Set this to allow updating of /refresh-guid/{REFRESH_KEY}
```

Comment:
ah, i forgot: you'll want to add `REFRESH__KEY` to the encrypted settings files for the staging and production site!